### PR TITLE
feat: add pictures to show what we mean with copyright & license header

### PR DESCRIPTION
**Pull request by kfenner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ The **F**ree and **O**pen **S**ource **S**oftware (FOSS) movement has created a
 
 **Headerless.org** therefore promotes to use **no license or copyright headers** in software project and use the centralized approaches to proudly state your FOSS license of choice. In case of a mixed license project, follow the **[12Factor principles](https://12factor.net/dependencies)**  and split your project up into isolated dependencies, each with its individual license.
 
+### What should I do:
+- Remove license and copyright headers from source code:
```
Comment (fh1ch) [inline README.md:12]: Missing new line afrer header

File: README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ The **F**ree and **O**pen **S**ource **S**oftware (FOSS) movement has created a
...
+![no copyright & license header](assets/no-header.png)
+
+- Remove also SPDX identifier from headers:
```
Comment (fh1ch) [inline README.md:15]: New line in a MD list not required (but can be added if design requires so).

**Comment by kfenner** [comment]
It's a bit in conflict with the chapter "Join the Movement" where also some steps are described.

**Comment by fh1ch** [comment]
@kfenner, no worries, I will incorporate your changes in the cirrent flow...

**Comment by fh1ch** [comment]
@kfenner I've took and added your images to the existing *ho to join* guide as part of PR #4.

Closing this one in the favour of PR #4.

**Pull request closed without merging**